cypress/e2e/dashboard/events/events-index.cy.ts

```diff
@@ -297,23 +297,50 @@ describe('events dashboard', () => {
 
   it('chapter admin should be allowed to edit event, but nobody else', () => {
     const eventId = 1;
+
+    cy.logout();
+    // newly registered user (without a chapter_users record)
+    cy.register();
+
+    cy.login(Cypress.env('JWT_TEST_USER'));
+    cy.reload();
+    cy.updateEvent(eventId, eventData).then(expectToBeRejected);
+    cy.logout();
+    // banned admin should be rejected
+    cy.login(Cypress.env('JWT_BANNED_ADMIN_USER'));
+    cy.reload();
+    cy.updateEvent(eventId, eventData).then(expectToBeRejected);
+
+    cy.logout();
     // admin of chapter 1
     cy.login(Cypress.env('JWT_ADMIN_USER'));
```

Comment:
```suggestion
    cy.login(Cypress.env('JWT_CHAPTER_1_ADMIN_USER'));
```
Same as the other login: it works because of the default login behaviour.